src/peripheral/itm.rs

```diff
@@ -174,42 +174,119 @@ pub struct ITMSettings {
     pub timestamp_clk_src: TimestampClkSrc,
 }
 
+/// Possible errors on [ITM::configure].
+#[derive(Debug, Eq, PartialEq, Copy, Clone)]
+pub enum ITMConfigurationError {
+    /// Global timestamp generation is not supported on this target.
+    /// Request [GlobalTimestampOptions::Disabled] instead.
+    ///
+    /// `ITM_TCR` register remains unchanged on this error.
```

Comment:
I think you can do the following to make it an intra-doc link:
```suggestion
    /// [`ITM_TCR`](struct@Tcr) register remains unchanged on this error.
```
And this can be extrapolated to the rest. I think it's also best to make the inner mono spaced with \`'s